src/main/kotlin/io/snyk/plugin/ui/toolwindow/VulnerabilityTreeCellRenderer.kt

```diff
@@ -106,8 +106,18 @@ class VulnerabilityTreeCellRenderer : ColoredTreeCellRenderer() {
                     issue.title
                 } + if (issue.ignored) " (ignored)" else ""
 
+                var obsolete = false
+                if (value.parent is IacFileTreeNode) {
+                    val parentTreeNode = value.parent as IacFileTreeNode
+                    val iacIssuesForFile = parentTreeNode.userObject as IacIssuesForFile
+                    if (iacIssuesForFile.obsolete) {
+                        text += obsoleteSuffix
```

Comment:
still mark issue as (obsolete), for other products only FileNode marked with that suffix and issues just grayed